pkg/adapterManager/manager.go

```diff
@@ -418,21 +420,29 @@ func (m *Manager) execute(ctx context.Context, cfg *cpb.Combined, requestBag, re
 		return status.WithError(fmt.Errorf("could not find aspect manager %#v", cfg.Aspect.Kind))
 	}
 
-	var builder adapter.Builder
-	if builder, found = m.builders.FindBuilder(cfg.Builder.Impl); !found {
+	var createAspect aspect.CreateAspectFunc
+	if builder, found := m.builders.FindBuilder(cfg.Builder.Impl); found {
+		var e error
+		createAspect, e = aspect.FromBuilder(builder, kind)
```

Comment:
Can't do that: e isn't pre-declared, and if we use `:=` it redeclares `createAspect` in a smaller scope.